### PR TITLE
Update collection-expression-arguments.md

### PR DESCRIPTION
**Pull request by CyrusNajmabadi**

### CODE REVIEW

File: proposals/collection-expression-arguments.md [hunk 1]
```diff
@@ -182,3 +182,18 @@ These forms seem to "read" reasonable well.  In all those cases, the code is "cr
 with the following 'args' to pass along to control the final instance, and then the subsequent elements used to
 populate it.  For example, the first line "creates a list of strings with a capacity 'arg' of two times the count
 of the values about to be spread into it"
+
+## Option 3: `new(...arguments...) [...elements...]`
+
+The design here would play off of how `new(...) { v1, v2, ... }` can already instantiate a target collection type and
+supply initial collection values.  The arguments in the `new(...)` clause would be passed to the constructor if
```
Comment (CyrusNajmabadi) [inline proposals/collection-expression-arguments.md:189]:
```suggestion

The design here would play off of how `new(...) { v1, v2, ... }` can already instantiate a target collection type
and supply initial collection values.  The arguments in the `new(...)` clause would be passed to the constructor if
```

**Comment by KennethHoff** [comment]
What about `MyList<int> listLike = (param1, param2)[1, 2, 3]`? It's very similar to 3, but without the ambiguity in terms of constructors.

It is probably ambiguous with casting though, right? 😅😞

**Comment by obonn1** [comment]
I understand and agree with the arguments about having any syntax outside of the `[]` but I think it's balanced by the concern that anything inside the brackets feels like it's an element of the collection. I don't really have a suggestion I love, but I think exploring other solutions outside of the brackets would be a good idea.

**Comment by CyrusNajmabadi** [comment]
> It is probably ambiguous with casting though, right?

Yes.  `(MyCollection)[a, b, c]` has meaning today, and is our *recommended* way to convert from a collection expr to a type.  That's a non-starter.

**Comment by CyrusNajmabadi** [comment]
>  but I think exploring other solutions outside of the brackets would be a good idea.

We'll explore any options people can suggest.  But we'll have to keep this sort of thing in mind.  FWIW, i don't view 'inside the brackets' as 'must be an element'.  It is simply the info controlling the collection.  That includes both the customization of the value and the elements within.

**Pull request merged**